typescript/web/src/pages/posts/[slug].tsx

```diff
@@ -19,6 +19,11 @@ import { CookieBanner } from "../../components/cookie-banner";
 
 const ChakraReactMarkdown = chakra(ReactMarkdown);
 
+const origin =
+  process.env.NEXTAUTH_URL ??
+  (process.env.VERCEL_URL && `https://${process.env.VERCEL_URL}`) ??
+  "https://labelflow.com";
```

Comment:
Can we make this value available globally (e.g. in a `src/constants.ts` file) under the name `ROOT_URL` or similar?

Reply:
Done in 4b37dbc44ebcd22d103db4ce6f887ee9e4df7cbc but I named it `WEB_APP_URL_ORIGIN` as it is not specific to articles